chore: add user name in get method

LeonelCeballos18 opened the pull request:
## What kind of PR is this?

- [x] Chore

## Description
-add `user.name.get.dto`
-add relation user and return user name

## Useful resources
<img width="544" alt="Screenshot 2025-04-07 at 16 56 53" src="https://github.com/user-attachments/assets/22ba5f94-cb70-49af-9596-3f062160074e" />

LeonelCeballos18 committed:
chore: add user name in get method

-add `user.name.get.dto`

cmagallon1 commented on src/services/appointment.service.ts:
```diff
@@ -61,16 +62,20 @@ export default class AppointmentService {
   }
 
-  async getById(id: number): Promise<Appointment> {
+  async getById(id: number): Promise<any> {
```
why any as type? 👀

cmoreno357 commented on src/services/appointment.service.ts:
```diff
@@ -47,9 +48,9 @@ export default class AppointmentService {
   ) {}
 
-  async get(): Promise<Appointment[]> {
+  async get(): Promise<any[]> {
```
Please type this properly

cmoreno357 reviewed (changes requested):
Please find a way to type things without using any.

mtopete1996 reviewed (commented):
Cant approve it until you solve the comments

LeonelCeballos18 committed:
Solving comments

-add `appointment dto` and added to index dto
-add `appointment response dto` and added to index dto
